Source/WebCore/editing/AlternativeTextController.cpp

```diff
@@ -676,6 +649,34 @@ void AlternativeTextController::applyAlternativeTextToRange(const SimpleRange& r
 
 #endif
 
+static inline void removeAutocorrectionIndictorMarkers(Document& document)
+{
+#if HAVE(AUTOCORRECTION_ENHANCEMENTS)
+    document.markers().dismissMarkers(DocumentMarker::CorrectionIndicator);
+#else
+    document.markers().removeMarkers(DocumentMarker::CorrectionIndicator);
+#endif
+}
+
+void AlternativeTextController::respondToAppliedEditing(Document& document, CompositeEditCommand* command)
+{
+#if USE(AUTOCORRECTION_PANEL)
+    if (command->isTopLevelCommand() && !command->shouldRetainAutocorrectionIndicator())
+        removeCorrectionIndicatorMarkers(m_document);
```

Comment:
Right?

```suggestion
        removeAutocorrectionIndictorMarkers(m_document);
```

Reply:
Nope, I just need to rename the method since I intentionally misspelled it at one point and forgot to change it back

Reply:
oh right yeah!